vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java

```diff
@@ -329,4 +330,45 @@ private void createCompositeLayout() {
 
         addCard("Using form layout inside a composite", layout);
     }
+    
+    private void createLayoutHandleColspans() {
+        // begin-source-example
+        // source-example-heading: Handling columns and colspans in a layout
+    	FormLayout colLayout = new FormLayout();
+    	//Setting the desired responsive steps for the columns in the layout
+    	colLayout.setResponsiveSteps(
+                new ResponsiveStep("0", 1),
+                new ResponsiveStep("21em", 2),
+                new ResponsiveStep("22em", 3));
+        TextField firstName = new TextField();
+        firstName.setPlaceholder("First Name");
+        TextField lastName = new TextField();
+        lastName.setPlaceholder("Last Name");
+        TextField email = new TextField();
+        email.setPlaceholder("Email");
+        TextField nickname = new TextField();
+        nickname.setPlaceholder("Username");
+        TextField website = new TextField();
+        website.setPlaceholder("Link to personal website");
+        TextField description = new TextField();
+        description.setPlaceholder("Enter a short description about yourself");
+        //Adding the components to it.
+        colLayout.add(firstName, lastName,  nickname, email, website); 
+        //you can set the desired column span for them individually.
+        colLayout.setColspan(website, 2);
+        //or just set it while adding them :
```

Comment:
```suggestion
        // you can set the desired column span for them individually.
```

Reply:
Woops missed the typo there. fixed in latest commit.